### PR TITLE
webgpu: fix notEqual error

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6669)

### CODE REVIEW

**Comment by gyagp** [comment]
Can we add some test cases to cover this?

File: tfjs-backend-webgpu/src/binary_op_util.ts [hunk 1]
```diff
@@ -109,8 +109,31 @@ const INT_DIV_VEC4 = `
   return vec4<f32>(resultTemp);
   `;
 
-const NOT_EQUAL = 'return f32(a != b);';
-const NOT_EQUAL_VEC4 = 'return vec4<f32>(a != b);';
+const NOT_EQUAL = `
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/binary_op_util.ts:112]: It seems that [webgl also doesn't check `isnan`](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/kernels/NotEqual.ts#L23). Can you verify whether webgl has the similar issue on Linux?

**Review by qjia7 — APPROVED** [review]
LGTM, thanks.

**Review by gyagp — APPROVED** [review]
LGTM. Offline talked and we already have unit test cases to cover this.

**Pull request merged**